js/jquery.cloudinary.js

```diff
@@ -1487,7 +1487,7 @@ var slice = [].slice,
       }
       expression = String(expression);
       operators = "\\|\\||>=|<=|&&|!=|>|=|<|/|-|\\+|\\*";
-      pattern = "((" + operators + ")(?=[ _])|" + Object.keys(Expression.PREDEFINED_VARS).join("|") + ")";
+      pattern = "((" + operators + ")(?=[ _])|(?<!\$)(" + Object.keys(Expression.PREDEFINED_VARS).join("|") + "))";
```

Comment:
We could. I feel like it might be less readable seeing as the string itself has so many parentheses already, and there are extra quotes in the content being concatenated.

I feel like the existing style is easier to grok in this specific case:

```javascript
pattern = "((" + operators + ")(?=[ _])|(?<!\$)(" + Object.keys(Expression.PREDEFINED_VARS).join("|") + "))";

pattern = `((${operators})(?=[ _])|(?<!\$)(${Object.keys(Expression.PREDEFINED_VARS).join("|")}))`;
```

Reply:
WDYT?